code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -328,7 +328,10 @@ There are several things that need to be remembered:
 		if(client && hud_used && hud_used.hud_shown)
 			client.screen += belt
 		update_observer_view(belt)
-		overlays_standing[BELT_LAYER] = belt.build_worn_icon(default_layer = BELT_LAYER, default_icon_file = 'icons/mob/clothing/belt.dmi')
+		var/t_state = belt.item_state
+		if(!t_state)
+			t_state = belt.icon_state
+		overlays_standing[BELT_LAYER] = mutable_appearance('icons/mob/clothing/belt.dmi', t_state, -BELT_LAYER)

```

Comment:
Explain the negative layer please?

Reply:
Honestly I just essentially copy pasted the code from the suit store layer and made it fit with belts. I've gone ahead and removed the negative layer since it doesn't seem to serve any function.